### ISSUE TITLE
Cannot read property 'key' of undefined

### ISSUE THREAD

**Issue by hasezoey** [discordjs/discord.js#2515]
**Please describe the problem you are having in as much detail as possible:**

**Further details:**

```
TypeError: Cannot read property 'key' of undefined
    at StreamDispatcher.createPacket (.\node_modules\discord.js\src\client\voice\dispatcher\StreamDispatcher.js:166:106)
    at StreamDispatcher.sendBuffer (.\node_modules\discord.js\src\client\voice\dispatcher\StreamDispatcher.js:134:25)
    at StreamDispatcher.process (.\node_modules\discord.js\src\client\voice\dispatcher\StreamDispatcher.js:243:14)
    at Socket.stream.once (.\node_modules\discord.js\src\client\voice\dispatcher\StreamDispatcher.js:324:12)
    at Object.onceWrapper (events.js:275:13)
    at Socket.emit (events.js:182:13)
    at emitReadable_ (_stream_readable.js:540:12)
    at process._tickCallback (internal/process/next_tick.js:174:19)
```

- discord.js version: 11.3.2 (latest npm)
- node.js version: 10.0.0
- Operating system: Windows 10 (latest)
- Priority this issue should have – please be realistic and elaborate if possible: 3 (1-10)

- [ ] I found this issue while running code on a __user account__

**Comment by hydrabolt** [comment]
Please tell us which version of discord.js you're using - this was fixed in master as far as I'm aware - and also we don't support user accounts anymore, so if your issue isn't reproducible on a bot account this issue will be closed.

**Comment by hasezoey** [comment]
(at the User account thing there is no "X")
Version: (updatetd)

**Comment by hydrabolt** [comment]
So this is already fixed in master, however I have now backported a fix to `11.3-dev` (installable with `npm i discordjs/discord.js#11.3-dev`); this issue will be closed whenever the new 11.x version is released.

**Comment by amishshah** [comment]
The fix has been released today in 11.4.
